sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1175,9 +1175,9 @@ _pi_queue::_pi_queue(std::vector<ze_command_queue_handle_t> &ComputeQueues,
         CopyQueueGroup.ImmCmdLists = std::vector<pi_command_list_ptr_t>(
             CopyQueueGroup.ZeQueues.size(), CommandListMap.end());
       }
-      CopyQueueGroupsByTID.insert({TID, CopyQueueGroup});
     }
   }
+  CopyQueueGroupsByTID.insert({TID, CopyQueueGroup});
```

Comment:
if I read the change correctly it additionally inserts empty copy-group if use of copy engines was disabled, right? Is that what was broken?

Reply:
Yes. Originally, the empty copy-group was always present as a normal, not per-thread data member.